backend/curriculum_tracking/tests/test_activity_log_entry_creators_envoked.py

```diff
@@ -106,3 +106,45 @@ def test_add_review(self):
             start_url, data={"status": COMPETENT, "comments": "blah more stuff"}
         )
         self.assertEqual(LogEntry.objects.count(), 2)
+
+
+class log_card_review_requested_Tests(APITestCase, APITestCaseMixin):
+    LIST_URL_NAME = "agilecard-list"  #based on class of api-viewset
+    SUPPRESS_TEST_POST_TO_CREATE = True
+    SUPPRESS_TEST_GET_LIST = True
+
+    def test_review_requested(self):
+        actor_user = UserFactory(is_superuser=True, is_staff=True)
+        content_item=factories.ProjectContentItemFactory(
+                project_submission_type=ContentItem.LINK, template_repo=None
+            )
+        card = factories.AgileCardFactory(
+            status=AgileCard.READY, 
+
+            
+            recruit_project=factories.RecruitProjectFactory(content_item=content_item),
+            content_item=content_item
+            
+        )
+        self.login(actor_user) #do we not login first before starting project?
+        card.start_project() #means card is now IP
+
+        request_review_url = f"{self.get_instance_url(card.id)}request_review/"
+        response = self.client.post(request_review_url)
+        assert response.status_code == 200, response.data
+
+        card.refresh_from_db()
+
+        # sanity check
+        assert card.assignees.count() == 1
```

Comment:
I am not sure why you check how many assignees the card has, is it relevant to check that in this test?

Reply:
haha, again this was me pattern matching from a previous test. Perhaps the sanity check was not meant to be kept in the code base? but I guess just a double check that all is well? I actually think I agree, that it is odd. Is there ever an occasion that a card would have more than one assignee....